code/game/mecha/working/ripley.dm

```diff
@@ -55,22 +67,55 @@
 
 /obj/mecha/working/ripley/update_desc()
 	. = ..()
-	if(!hides) // Just in case if hides are somehow removed
+	if(!hides && !plates && !drake_hides) // Just in case if armor is removed
 		desc = initial(desc)
 		return
-	if(hides == 3)
+
+	// Goliath hides
+	if(hides == HIDES_COVERED_FULL)
 		desc = "Autonomous Power Loader Unit. It's wearing a fearsome carapace entirely composed of goliath hide plates - its pilot must be an experienced monster hunter."
 	else
 		desc = "Autonomous Power Loader Unit. Its armour is enhanced with some goliath hide plates."
 
+	// Metal plates
+	if(plates == PLATES_COVERED_FULL)
+		desc = "Autonomous Power Loader Unit. Its armor is completely lined with metal plating."
+	else
+		desc = "Autonomous Power Loader Unit. Its armor is reinforced with some metal plating."
```

Comment:
This will cause the description to say that it has some metal plating, if it doesn't have any hides or plating at all

Reply:
Goliath hides never got broken like what you say, so clear copypasta for metal plating are NOT affect this too. 
This description will change only if you have any number or plate\hide on mech already.